### DISCUSSION TITLE
请问考不考虑在Greasy Fork、GitLab或者Gitee上面发布这个脚本？

### DISCUSSION THREAD

**Discussion by GeniusLv2006** [the1812/Bilibili-Evolved#3651] [提问 / Q&A]
Readme文档里面也有提到，国内网络访问GitHub实在不怎么方便，请问考不考虑在Greasy Fork、GitLab或者Gitee发布一下呢？这样做首先脚本更新就不会出问题了，其次就是Greasy Fork出现过盗版这个脚本的现象，那么官方直接把脚本发布到Greasy Fork上面不是更好吗？
谢谢。

**Comment by the1812** [comment]
网络不方便的话, 可以自己改一版指向其他站点: https://github.com/the1812/Bilibili-Evolved/blob/master/doc/cdn.md

GreasyFork 因为安装太过简单 (安装按钮放在了使用说明的前面), 会带来很多麻烦, 因此不会在 GreasyFork 上发布.
